README.md

```diff
@@ -87,6 +87,8 @@ We are providing example apps for Swift and Objective-C. These apps demonstrate
 
 Gini Vision Library can either be installed by using CocoaPods or by manually dragging the required files to your project.
 
+**Note on iOS 10:** No matter which option you choose if you want to support iOS 10 you need to specify the `NSPhotoLibraryUsageDescription` key in your `Info.plist` file. And if you're using the [Gini iOS SDK](https://github.com/gini/gini-sdk-ios) you need to add support for "Keychain Sharing" in your entitlements by adding a `keychain-access-groups` value to your entitlements file.
```

Comment:
Whoaaa...you are right! I completely messed up the keys. We only need `NSCameraUsageDescription` and `NSPhotoLibraryUsageDescription ` is only for development. Will fix that and add a sentence on why we need it.